### PR TITLE
Login ui improvement

### LINKED ISSUE #716: Create a better UI for login
Currently, the UI for login is very simple and does not look so attractive.
When the user clicks on the login option, it must appear at the middle of the page as a box and must have better designs. 
Something similar to Spectral Workbench and Plots2.

Thanks!

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #716  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

Selection_186

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by divyabaid16** [comment]
@publiclab/reviewers

**Comment by IshaGupta18** [comment]
Great work here! I think this is good to go!

File: app/views/layouts/_login.html.erb [hunk 1]
```diff
@@ -13,9 +13,27 @@
...
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">Log In</h4>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <%= render :partial => "layouts/login_form" %>
```
Comment (alaxalves) [inline app/views/layouts/_login.html.erb]:
```suggestion
            <%= render partial: "layouts/login_form" %>
```

Comment (alaxalves) [reply app/views/layouts/_login.html.erb]: Just using the new Ruby sintax ;)

**Comment by CoderJolly** [comment]
Awesome Login Modal ! Its ready to go !! 🎉

**Comment by jywarren** [comment]
Fantastic, thank you!

**Pull request merged**